homeassistant/components/binary_sensor/danfoss_air.py

```diff
@@ -0,0 +1,60 @@
+"""
+Binary sensors for Danfoss Air HRV.
```

Comment:
Normal convention has these start with `Support for...`.

In this case, it would be something like: `Support for the Danfoss Air HRV binary sensor platform.`